zeebe-job-worker/src/main/kotlin/job/subprocess/ShipParcel.kt

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Zeebe Community License 1.1. You may not use this file
+ * except in compliance with the Zeebe Community License 1.1.
+ */
+package job.subprocess
+
+import io.camunda.zeebe.client.ZeebeClient
+import io.camunda.zeebe.client.ZeebeClientBuilder
+import io.camunda.zeebe.client.api.response.ActivatedJob
+import io.camunda.zeebe.client.api.worker.JobClient
+import io.camunda.zeebe.client.api.worker.JobHandler
+import java.time.Duration
+import java.util.Scanner
+
+object ShipParcel {
+    @JvmStatic
+    fun main(args: Array<String>) {
+        val defaultAddress = "localhost:26500"
+        val envVarAddress = System.getenv("ZEEBE_ADDRESS")
+        val clientBuilder: ZeebeClientBuilder = ZeebeClient
+            .newClientBuilder()
+            .gatewayAddress(defaultAddress)
+            .usePlaintext()
+
+        val jobType = "ship-parcel"
+        clientBuilder.build().use { client ->
+            println("Opening job worker for shipping parcel.")
+            client
+                .newWorker()
+                .jobType(jobType)
+                .handler(ShipParcelHandler())
+                .timeout(Duration.ofSeconds(10))
+                .open().use {
+                    println("Shipping parcel job worker is opened.")
+                    waitUntilSystemInput("exit")
+                }
+        }
+    }
+
+    private fun waitUntilSystemInput(exitCode: String) {
```

Comment:
Also there's a ton of code in each of these workers that could just be removed if you only have one main function